package.json

```diff
@@ -83,6 +83,7 @@
     "ts-migration:dashboard:deploy": "gh-pages --dist development/ts-migration-dashboard/build --remote ts-migration-dashboard"
   },
   "resolutions": {
+    "**/@babel/parser": "7.16.4",
```

Comment:
The newest version of depcheck (`v1.4.3`) pins `@babel/parser` to this version, so we don't need to use a resolution anymore (See https://github.com/depcheck/depcheck/blob/1.4.3/package.json#L52)

Reply:
Gotcha. Will put this back in draft, again, until I have this updated 😅

Reply:
Hey, sorry this ended up being a problem on the release branch as well, so I went ahead and made a separate PR for this: #15690